frontend/components/Goals/index.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import image from './images/Squiggles Monochromatic.png';
```

Comment:
don't forgot to delete the data used for demo and use the demo content in the storybook